modules/administration-guide/attachments/migration/1-prepare.sh

```diff
@@ -4,23 +4,24 @@ set -o nounset
 set -o pipefail
 
 K8S_CLI=${K8S_CLI:-oc}                                                           # {orch-cli}
-PRODUCT_DEPLOYMENT_NAME=${PRODUCT_DEPLOYMENT_NAME:-che}                          # {prod-deployment}
 PRODUCT_ID=${PRODUCT_ID:-eclipse-che}                                            # {prod-id}
 INSTALLATION_NAMESPACE=${INSTALLATION_NAMESPACE:-eclipse-che}                    # {prod-namespace}
-CHE_CLUSTER_CR_NAME=${CHE_CLUSTER_CR_NAME:-eclipse-che}                          # {prod-checluster}
-IDENTITY_PROVIDER_DEPLOYMENT_NAME=${IDENTITY_PROVIDER_DEPLOYMENT_NAME:-keycloak} # {identity-provider-id}
+
+MIGRATION_PRODUCT_DEPLOYMENT_NAME=${MIGRATION_PRODUCT_DEPLOYMENT_NAME:-che}      # {prod-migration-deployment}
```

Comment:
Same remark as above. Probably this is makes things clearer:
```suggestion
PRE_MIGRATION_PRODUCT_DEPLOYMENT_NAME=${PRE_MIGRATION_PRODUCT_DEPLOYMENT_NAME:-che}      # {prod-migration-deployment}
```